README.md

```diff
@@ -108,73 +116,58 @@ In these environments, two problems can be encountered, the solution for which i
 __NOTE:__ Some restrictions will also apply on the version of C and C++ standard
 libraries installed on the system. This needs further investigation.
 
-Quick Install
-------------
-
-### Building Ginkgo
+Building and Installing Ginkgo
+---------------------------------
 
 To build Ginkgo, you can use the standard CMake procedure.
 
 ```sh
 mkdir build; cd build
 cmake -G "Unix Makefiles" .. && make
+make install # will install to the system default location
 ```
 
 By default, `GINKGO_BUILD_REFERENCE` is enabled. You should be able to run
 examples with this executor. By default, Ginkgo tries to enable the relevant
 modules depending on your machine environment (present of CUDA, ...). You can
 also explicitly compile with the OpenMP, CUDA, HIP or DPC++(SYCL) modules enabled to
-run the examples with these executors. Please refer to the [Installation
-page](./INSTALL.md) for more details.
+run the examples with these executors.
 
-After the installation, CMake can find ginkgo with `find_package(Ginkgo)`.
-An example can be found in the [`test_install`](test/test_install/CMakeLists.txt).
+Please refer to the [Installation page](./INSTALL.md) for more details.
 
-### Ginkgo Examples
+Tip: After installation, in your CMake project, Ginkgo can be added with `find_package(Ginkgo)`.
+An example can be found in the [`test_install`](test/test_install/CMakeLists.txt).
```

Comment:
Maybe mention that we export the target `Ginkgo::ginkgo`. I always find it helpful to know the actual target that I'm going to use, instead of guessing what the name might be.